### PR TITLE
Fix issue #143, hide button and show success message, values sidebars

### PR DESCRIPTION
**Pull request by schalkneethling**
@chrismore r?

### CODE REVIEW

File: data/css/all-aboard.css [hunk 1]
```diff
@@ -219,6 +219,12 @@ a:focus {
     display: inline-block;
 }
 
+.token-received-msg {
```
Comment (chrismore) [inline data/css/all-aboard.css]: @schalkneethling can you make this font normal instead of bold and slightly smaller? probaby 1.0rem instead of 1.2.

**Comment by chrismore** [comment]
@schalkneethling code is looking good. I made a comment in the code about the css for the here's your badge.

Also, @bniolet1 is dropping in a link to the in-product copy for copy specific to just the values track when you answer the question. There will be specific follow up copy below the answers and above the "here's your badge" in normal body copy font. They should just be in the html and depending on your answer hides/shows one of the funny follow up copy sentences.

**Comment by chrismore** [comment]
@schalkneethling focus this PR on just the follow up copy for the values sidebar and use issue #166 for the "Here's your badge" message.

**Comment by schalkneethling** [comment]
@chrismore This has been updated. We did not address what should happen if the user makes no selection so, currently I simply fallback to the correct answer's response copy if no selection was made.

**Comment by chrismore** [comment]
@schalkneethling all looks good. can you just make the style of the follow up copy to be italics?

**Comment by chrismore** [comment]
perfect. ship it!

**Pull request merged**